specs/lib/gltfPipelineSpec.js

```diff
@@ -120,7 +120,7 @@ describe('gltfPipeline', function() {
         readGltf(gltfEmbeddedPath, options, function (gltf) {
             var initialUri = gltf.buffers.CesiumTexturedBoxTest.uri;
             processJSON(gltf, options, function () {
-                var finalUri = gltf.buffers.CesiumTexturedBoxTest.uri;
+                var finalUri = gltf.buffers.uri;
```

Comment:
Since in this test, there can only be one buffer (I assume). Just get the first buffer:
```javascript
var firstBufferId = Object.keys(gltf.buffers)[0];
var testBuffer = gltf.buffers[firstBufferId];
```